### ISSUE TITLE
"any shape or size" is inaccurate, implies this is a big data tool

### ISSUE THREAD

**Issue by simonw** [simonw/datasette.io#109]
Pointed out on Hacker News: https://news.ycombinator.com/item?id=31531718

Here's the intro text on https://datasette.io/ right now:

> Datasette is a tool for exploring and publishing data. It helps people take data of any shape or size, analyze and explore it, and publish it as an interactive website and accompanying API.

I agree: "any shape" does imply that this is a big data tool, when actually there are points where your data gets too big for Datasette and other tools would be more appropriate.

**Comment by simonw** [comment]
Had some fun feeding this through GPT3. My prompts in bold:

> **Reword this to remove the note about any size: Datasette is a tool for exploring and publishing data. It helps people take data of any shape or size, analyze and explore it, and publish it as an interactive website and accompanying API.**
> 
> Datasette is a tool for exploring and publishing data. It helps people take data, analyze and explore it, and publish it as an interactive website and accompanying API.
> 
> **Reword this to remove the note about any size, then add a lot more excitement: Datasette is a tool for exploring and publishing data. It helps people take data of any shape or size, analyze and explore it, and publish it as an interactive website and accompanying API.**
> 
> Datasette is a powerful tool for exploring and publishing data. It helps people take data of any shape or size, analyze and explore it, and publish it as an interactive website and accompanying API. Datasette makes it easy to work with data, whether it's small and simple or large and complex. With Datasette, you can quickly turn your data into valuable insights that can be shared with others.

I don't think I'll use these though.

**Comment by simonw** [comment]
I'd quite like to get data cleaning in there:

> Datasette is a tool for exploring and publishing data. It helps people import and clean data, analyze and explore it, and publish it as an interactive website and accompanying API.

It's not strictly accurate though: you can clean data using the Datasette suite of tools (see the forthcoming tutorial, #108) but it's not really a solution for data cleaning out of the box yet.

**Comment by simonw** [comment]
I'm just going to drop "or size" but keep "any shape".

**Datasette is a tool for exploring and publishing data. It helps people take data of any shape, analyze and explore it, and publish it as an interactive website and accompanying API.**

**Comment by westurner** [comment]
- "Any thoughts/future plans for using DuckDB? #968" 
 https://github.com/simonw/datasette/discussions/968
